content/en/blog/identity-hack-ssi-introduction/index.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2023-05-19
+title: "Digital Identity Hack Part 3 – Introduction to SSI presentation"
+linkTitle: "Digital Identity Hack Part 3 – Introduction to SSI presentation"
+description: "In this blog, we discuss about the Digital Identity Hack event, which focused on unlocking the potential of self-sovereign identity technologies in revolutionizing the financial landscape."
+author: Pekka Kaipio
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+The rise of **self-sovereign identity** (SSI) presents a groundbreaking
+opportunity for progress and expansion in the financial sector, with the
+capacity to reshape digital financial services, simplify transactions, and
+facilitate seamless interactions. To capitalize on this potential, OP Lab and
+the EU funded Powered by Blockchain project organized a seminar day and online
+hackathon, exploring the innovative power of SSI technologies and institutional
+decentralized finance, setting the stage for **a safer, more inclusive, and
+efficient worldwide financial ecosystem.** 
+
+At the seminar event, participants engaged in a panel discussion centered on
+blockchain projects and **a presentation delving into self-sovereign identity**.
+In the following section, we will examine the essential insights and highlights
+that emerged from the SSI presentation.
+
+## Introduction to Self-Sovereign Identity
+
+{{< youtube-wrapper utNq_IsH_Y4 939px 649px >}} <em>Harri Lainio's presentation:
+Introduction to Self-Sovereign Identity</em>
+
+In the event day presentation on Self-Sovereign Identity, **Harri Lainio from OP
+Lab** introduced SSI, its potential impact on the world, and Findy Agency
+open-source solution for implementing it. The speaker highlighted the power
+dynamics of the current digital identity model, which has centralized power in
+the hands of a few online platforms. In the next evolution of the internet,
+**Web 3, aims to shift power back to individuals** and things, thus empowering
+users to take control of their digital identities. 
+
+Learn more about Findy Agency [here](https://findy-network.github.io/).
+
+### Take Control of your Identity 
+
+**SSI offers the promise of an internet where individuals are sovereign,
+building their own networks, and serving as the roots of these networks.** By
+examining the incentives for SSI, the speaker noted that humans are cooperative
+but inherently distrustful, making the exchange of value challenging.
+Institutions like banks and governments have historically reduced uncertainty,
+but the speaker questioned whether there are alternative ways to achieve this.
+He acknowledged that blockchain technology has solved some issues, such as
+double spending, but still has a long way to go. 
+
+The speaker emphasized that SSI is about putting the user at the center of their
+digital identity: ***"I want to be in the center. And that's something that SSI
+promises to handle."*** By empowering individuals to take control of their
```

Comment:
By emp... -> By empowering individuals to take control of their digital selves, SSI can reshape the internet and transform how people interact and exchange value online.